workshops/README.md

```diff
@@ -2,17 +2,17 @@
 
 The workshops are a collection of assignments for learning how to automate with Ansible. The workshops are introduced by the decks and are assigned to the students during the presentations.
 
-## How to use those
+The workshops are set up as exercises to be done by the participants, and are most suitable for smaller audiences.
```

Comment:
Again, clarifying the situation around the workshops to better guide the readers.